docs/proposals/ldap-group-sync.md

```diff
@@ -0,0 +1,92 @@
+# Allowing Groups to be Based on and Synced With External LDAP Records
```

Comment:
I think we want to enable group/member query specifications that allow the two main classes of schemas. Examples for the two main use cases would be good, but I don't think we'll target precanned support for specific schemas.